.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+## **SUBMISSIONS LACKING A RESPONSE TO ONE OF THE FIELDS WILL BE DELETED**
+
+### Pull Request Description:
+
+> Write a short description about the Pull Request
```

Comment:
This pull request template is poor. You have not covered all kinds of pull requests.